testcase/cross-region/cross-region.go

```diff
@@ -34,6 +34,8 @@ add placement policy
 	role=leader
 	replicas=1`
 
+var DCLocations = []string{}
```

Comment:
If the number of DCs must be 3, then the type of `DCLocations` can be `[3]string`.

Reply:
I think the number of dc(s) may change in future, use slice here is more flexible.